src/locales/en/translation.json

```diff
@@ -16,93 +16,114 @@
       "fundingIncrease": "You drew a funding card that can\ncontribute towards the implementation of a measure!",
       "funding": {
         "name": "Funding",
-        "description": "This card must be paired with a measure card to allow the measure card to be implemented."
+        "description": "This card must be paired with a measure card to allow the measure card to be implemented.",
+        "eventDescription": "This card must be paired with a measure card to allow the measure card to be implemented."
       },
       "hazardousSubstanceRegulation": {
         "name": "Hazardous substance regulation",
-        "description": "You can pair a producer and an animal sharing the same habitat symbol to secure a habitat-tile"
+        "description": "You can pair a producer and an animal sharing the same habitat symbol to secure a habitat-tile",
```

Comment:
BTW we should move commandBar string for this card into translation file as well.

Reply:
Agreed, done